### ISSUE TITLE
[quality][performance] Format the performance numbers in the log

### ISSUE THREAD

**Issue by kittaakos** [eclipse-theia/theia#6531]
### Description
 

Instead of showing
```
root WARN EditorNavigationContribution.onStart is slow, took: 421.90000000118744 ms
root INFO Changed application state from 'init' to 'started_contributions'.
root INFO Changed application state from 'started_contributions' to 'attached_shell'.
root INFO [49068a12-89ed-4ccd-9892-dc28f7ad5ca1] Sync of 0 plugins took: 206.99999999487773 ms
root INFO Changed application state from 'attached_shell' to 'initialized_layout'.
root INFO [49068a12-89ed-4ccd-9892-dc28f7ad5ca1] Load contributions of 0 plugins took: 0.09999999747378752 ms
root INFO [49068a12-89ed-4ccd-9892-dc28f7ad5ca1] Start of 0 plugins took: 46.29999999451684 ms
root INFO [49068a12-89ed-4ccd-9892-dc28f7ad5ca1] Sync of 0 plugins took: 6.600000000617001 ms
root INFO [49068a12-89ed-4ccd-9892-dc28f7ad5ca1] Load contributions of 0 plugins took: 0 ms
root INFO [49068a12-89ed-4ccd-9892-dc28f7ad5ca1] Start of 0 plugins took: 18.799999998009298 ms
```

I propose formatting the performance numbers, so it is easier to read:
```
root WARN EditorNavigationContribution.onStart is slow, took: 421.9 ms
root INFO Changed application state from 'init' to 'started_contributions'.
root INFO Changed application state from 'started_contributions' to 'attached_shell'.
root INFO [49068a12-89ed-4ccd-9892-dc28f7ad5ca1] Sync of 0 plugins took: 207 ms
root INFO Changed application state from 'attached_shell' to 'initialized_layout'.
root INFO [49068a12-89ed-4ccd-9892-dc28f7ad5ca1] Load contributions of 0 plugins took: 0.01 ms
root INFO [49068a12-89ed-4ccd-9892-dc28f7ad5ca1] Start of 0 plugins took: 46.3 ms
root INFO [49068a12-89ed-4ccd-9892-dc28f7ad5ca1] Sync of 0 plugins took: 6.6 ms
root INFO [49068a12-89ed-4ccd-9892-dc28f7ad5ca1] Load contributions of 0 plugins took: 0 ms
root INFO [49068a12-89ed-4ccd-9892-dc28f7ad5ca1] Start of 0 plugins took: 18.8 ms
```

### Reproduction Steps

**OS and Theia version:**

**Diagnostics:**

**Comment by vince-fugnitto** [comment]
@kaiyue0329 this is a fairly simple change if you'd like to fix it :)

**Comment by vince-fugnitto** [comment]
Closed by #6858
